server/odc-server/src/main/java/com/oceanbase/odc/server/config/MvcConfiguration.java

```diff
@@ -95,7 +96,7 @@ public void addCorsMappings(CorsRegistry registry) {
 
     @Override
     public void configureContentNegotiation(ContentNegotiationConfigurer config) {
-        config.favorPathExtension(false);
```

Comment:
why remove `config.favorPathExtension(false);` ?

Reply:
Because the `favorPathExtension` config is default set to `false`, so i remove it.
By the way, it has been marked `Deprecated`.